app/models/like.rb

```diff
@@ -0,0 +1,10 @@
+class Like < ApplicationRecord
+  belongs_to :post
+  belongs_to :user
+
+  after_save :update_likes_counter
```

Comment:
- [ ]  Please make your update_likes_counter method to be set to private, which will make your class to be called only within the like class 👍🏿